onnxruntime/python/tools/transformers/shape_infer_helper.py

```diff
@@ -0,0 +1,66 @@
+import os
+import sys
+
+# In ORT Package the symbolic_shape_infer.py is in ../tools
+if os.path.exists("../tools/symbolic_shape_infer.py"):
+    sys.path.append(os.path.join(os.path.dirname(__file__), '../tools'))
+else:
+    sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from symbolic_shape_infer import *
+
+
+class SymbolicShapeInferenceHelper(SymbolicShapeInference):
+    def __init__(
+            self,
+            model,
+            dynamic_axis_mapping={
+                "batch_size": 4,
```

Comment:
what happen when user's symbolic dimension is not named as "batch_size" & "seq_len"? For example, in GPT-2, there is past_seq_len etc.

Reply:
if there's other names we need to pass them to ctor(from the current design)

Reply:
actually GPT-2 works with only "batch_size" & "seq_len".